### DISCUSSION TITLE
npm cli development frozen?

### DISCUSSION THREAD

**Discussion by mariusa** [npm/feedback#642] [CLI] [answered]
The [roadmap](https://github.com/npm/roadmap/projects/1) shows only one development for npm in the past year (workspaces part 2), and no new plans.
In the meantime, projects are switching to yarn/pnpm for reasons such as `yarn is quicker overall and the lockfile is nicer.`

Do you plan to keep evolving npm, or just focus on the registry?

**Comment by ljharb** [comment]
Overrides was also released in that time. You may want to check the npm/rfcs repo; there’s a lot going on.

**Comment by darcyclarke** [comment, marked as answer]
@mariusa CLI is in active development - apologize we haven't shared more about our roadmap towards v9 & beyond (that public roadmap is definitely out of date) but we're definitely still developing & maintaining the project. As @ljharb noted, a lot of our planning/feature development happens in the [npm/rfcs](https://github.com/npm/rfcs) repo. Expect to see/hear more in the coming months.
